### PR TITLE
Automatically convert `:has` to `:-abp-has` on filters build

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/FiltersRegistry/issues/662

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,4 +30,57 @@ if (whitelist.length > 0 || blacklist.length > 0) {
...
-compiler.compile(filtersDir, logPath, reportPath, platformsPath, whitelist, blacklist, customPlatformsConfig);
+/**
+ * Search for specific files by extension type and
+ * replaces all occurrencies of necessary strings to new ones in the build results.
+ */
+
+ async function replaceInFilters() {
+/**
+ *  
+ * @param {String} dir - specific directory that limits the search.
```
Comment (slavaleleka) [inline index.js:41]: `string` should be lower-case 

`searchFiles()` should also have a description

File: index.js [hunk 2]
```diff
@@ -29,4 +30,57 @@ if (whitelist.length > 0 || blacklist.length > 0) {
...
+        platformsPath,
+        whitelist,
+        blacklist,
+        customPlatformsConfig,
+    );
+    // TODO: remove this when ExtCSS v2.0 is released to all AG products.
+    await replaceInFilters();
+}
+
+main();
```
Comment (ameshkov) [inline index.js:86]: Add a new line

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (zloyden) [reply index.js]: Done.

File: index.js (hunk 2 above)
Comment (zloyden) [reply index.js]: Also done.

**Review by ameshkov — APPROVED** [review]

**Pull request merged**